pgextwlist.c

```diff
@@ -415,14 +425,61 @@ call_ProcessUtility(PROCESS_UTILITY_PROTO_ARGS,
 						   | SECURITY_RESTRICTED_OPERATION);
 
 	if (action)
-		call_extension_scripts(name, schema, action,
-							   "before", old_version, new_version);
+	{
+		/* "drop extension" can list several extensions, walk them here */
+		if (!strcmp(action, "drop"))
```

Comment:
I prefer using `if (strcmp(action, "drop") == 0)` and keep the ! operator for things that convey a sense of being false, or unsuccessful.